templates/stamp/stamp-v2.json

```diff
@@ -0,0 +1,1174 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
```

Comment:
Yes. What we have is a break up of deployment components due to dependency management. 

1. First, we need to provision keyvault & populate it before kicking off the main deployment. Unfortunately, arm doesn't support populating the values locally. Therefore, the spin off from the main deployment

2. We now provision storage that is used in the backup flow. The storage account is dynamically provisioned based on the cluster name (previously, we created the account separately and checked in the name and key). Problem is azure listkey has very restrictive use (in terms of where it can be placed).

Now, these conditions necessitated the 2 part deployment. If you look at the deployments, you will see that they are using template links. As a P2, work could be done to remove the already deployed components from the main stamp-v2 template. That will change the parameters but require an unspecified amount of template refactor (references). So this is a mid-way compromise.